src/server/saved_objects/service/lib/repository.js

```diff
@@ -228,6 +239,10 @@ export class SavedObjectsRepository {
    * @returns {promise}
    */
   async delete(type, id, options = {}) {
```

Comment:
Is it safe to assume that we want `deleteByNamespace` to ignore the `allowedTypes`, and just delete everything?

Reply:
This is a good question. I'm leaning towards allowing it to be across all defined types within a namespace.